docs/1.0.0/configuration.md

```diff
@@ -0,0 +1,82 @@
+# Configuration
+
+```hcl
+component "velero" {
+  namespace = "velero"
+
+  # Params default values
+
+  aws = {
+    eksRole = "" # optional, IAM Role ARN for EKS RBAC
+    iamRole = "" # optional, IAM Role ARN for deploying
+  }
+
+  backups = {
+    provider = "" # required, storage provider for backups, available: "aws"
+                  # other providers in https://velero.io/docs/v1.5/supported-providers/ might work but haven't been tested
+
+    # if using aws for storage, this section is required and must point to an S3 bucket
+    s3 = {
+      region = ""          # required, bucket region
+      endpoint = ""        # required, S3 endpoint
+      pathStyle = false    # required, enable to force path-style requests (eg. minio and similar providers)
+      insecure = false     # required, enable to skip server certificate validation
+      encrypted = true     # required, enable to use server-side encryption
+      bucket = ""          # required, bucket name
+      prefix = ""          # optional, prefix/directory to upload backups into
+      caCert = ""          # optional, base64 encoded CA bundle to be used for verifying TLS connections
+      accessKeySecret = "" # optional, AWS Access key ID to access the bucket (overrides credentialsSecret/credentialsPath)
```

Comment:
`accessKeySecret` and `secretKeySecret` are object with a `name` and `key` field each

example
```hcl
accessKeySecret = {
  name = "name-of-the-secret"
  key = "accessKey"
}
```